### PR TITLE
Login page written but not connected to app or auth

### PR DESCRIPTION
**Pull request by tarrinneal**

### CODE REVIEW

File: ui/lib/login/widget/login_form.dart [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+
+class LoginForm extends StatefulWidget {
+  @override
+  LoginFormState createState() {
```
Comment (GikuyuNderitu) [inline ui/lib/login/widget/login_form.dart]: nit: LoginFormState createState() => LoginFormState();

File: ui/lib/login/widget/login_form.dart [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      child: FittedBox(
+        alignment: Alignment.center,
+        fit: BoxFit.contain,
+        child: Form(
+            child: Column(
+          children: <Widget>[
+            TextFormField(
+              validator: (String? value) {
+                return (value != null &&
+                        RegExp(r'^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$')
```
Comment (GikuyuNderitu) [inline ui/lib/login/widget/login_form.dart]: Move to topLevel final var then use inline eg.:
```dart
final emailRegex = RegExp(//...
```

File: ui/lib/login/page/login_page.dart [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    return Scaffold(
+      body: Stack(
+        children: <Widget>[
+          SvgPicture.asset(
+            assetName,
+            semanticsLabel: 'Harmony Logo',
+            fit: BoxFit.scaleDown,
+            height: Sizing.serverSideBarWidth,
+          ),
+          Container(
```
Comment (GikuyuNderitu) [inline ui/lib/login/page/login_page.dart]: Remove container since it's not doing anything

**Review by GikuyuNderitu — CHANGES_REQUESTED** [review]

File: ui/lib/login/widget/login_form.dart (hunk 1 above)
Comment (tarrinneal) [reply ui/lib/login/widget/login_form.dart:10]: I pulled this from the flutter docs, is this not best practice?

**Pull request closed without merging**